cloud-haven-react-app/src/components/VendorAppPage/VendorAppPage.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { CustomForm } from '../components';
+import { Container } from 'react-bootstrap';
+import './VendorAppPage.css';
+
+export default props => {
+    return (
+        <Container>
+            <h1>
+                {props.vendorId}
+            </h1>
+            <CustomForm fields={props.components[0].Fields}/>
```

Comment:
is there an agreement that the first element would be the form?

Reply:
No there is no agreement yet. I just put that there because we're only using dummy data for now. Once we're using the actual vendor app API we can change this.